crates/eips/src/eip4844/mod.rs

```diff
@@ -86,9 +91,146 @@ pub const BYTES_PER_COMMITMENT: usize = 48;
 
 /// How many bytes are in a proof
 pub const BYTES_PER_PROOF: usize = 48;
+/// A fixed-size container for binary data, designed to hold exactly 131,072 bytes.
+#[derive(Clone, Debug, RlpEncodable, RlpDecodable, Hash, PartialEq, Eq)]
```

Comment:
RlpDecodable we can't derive but must impl manually because we also need to enforce length